dss/subscriptions_v2/__init__.py

```diff
@@ -16,11 +16,25 @@ class SubscriptionData:
     FORM_FIELDS = 'form_fields'
     PAYLOAD_FORM_FIELD = 'payload_form_field'
     ATTACHMENTS = 'attachments'
+    STATS = 'stats'
+    BLANK_STATS = {"attempted": 0, "successful": 0, "failed": 0}
```

Comment:
I'm not sure ``BLANK_STATS` should be a part of `SubscriptionData` since it implies that it is a field.  I would have it as a separate var outside of the class.

Reply:
Moved